### ISSUE TITLE
Reordering clips in a new (unsaved) message with keyboard navigation doesn't work

### ISSUE THREAD

**Issue by robby1066** [robby1066/keepposted-help#78]
# Describe the bug
When using keyboard navigation to reorder the clips in a brand new message, 

## Steps To Reproduce
1. Go to compose, choose a message type that generates suggested clips (prepare for a conversation, for example)
2. Use the keyboard to tab into the re-ordering UI
3. Try to move clips around
4. Observe that moving elements up doesn't work as expected

## Expected behavior
A person using keyboard navigation should be able to arrange the clips as expected.

## Screenshots
https://user-images.githubusercontent.com/342510/127534831-2583b72d-b35b-4806-80be-bade65fab8cf.mp4

## Additional context
This does _not_ appear to be a problem once the message is saved. It only shows up on the new message UI

**Comment by robby1066** [comment]
This was because a sortable-index wasn't being added to new clips that weren't saved yet. Fixed and deploying today.

**Comment by robby1066** [comment]
Fixed and tested in production. Closing
